tracer/build/_build/Build.GitHub.cs

```diff
@@ -1120,48 +1088,80 @@ async Task HideCommentsInPullRequest(int prNumber, string prefix)
                        .PullRequest(prNumber)
                        .Comments()
                        .AllPages()
-                       .Select(issue => new { issue.Id, issue.Body, issue.IsMinimized, });
-
-            var issueComments =  (await connection.Run(query)).ToList();
+                       .Select(comment => new { comment.Id, comment.Body });
 
-            Console.WriteLine($"Found {issueComments} comments for PR {prNumber}");
+            var prComments = (await connection.Run(query)).ToList();
+        
+            Console.WriteLine($"Found {prComments.Count} comments for PR {prNumber}");
 
-            var count = 0;
-            foreach (var issueComment in issueComments)
+            var updated = false;
+            foreach (var prComment in prComments)
             {
-                if (issueComment.IsMinimized || ! issueComment.Body.StartsWith(prefix))
+                if (! prComment.Body.StartsWith(title))
                 {
                     continue;
                 }
 
                 try
                 {
-                    var arg = new MinimizeCommentInput
+                    var arg = new UpdateIssueCommentInput
                     {
-                        Classifier = ReportedContentClassifiers.Outdated,
-                        SubjectId = issueComment.Id,
+                        Id = prComment.Id,
+                        Body = markdown,
                         ClientMutationId = clientId
                     };
 
                     var mutation = new Mutation()
-                                  .MinimizeComment(arg)
-                                  .Select(x => new { x.MinimizedComment.IsMinimized });
+                                  .UpdateIssueComment(arg)
+                                  .Select(x => new { x.IssueComment.Id });
 
                     await connection.Run(mutation);
-                    count++;
+                    updated = true;
+                    Console.WriteLine($"Updated comment {prComment.Id} for PR {prNumber}");
```

Comment:
If you do decide to go with the suggestion above:
```suggestion
                    Console.WriteLine($"Updated comment {prComment.Id} for PR {prNumber}");
                    break;
```